cmd/check/root.go

```diff
@@ -1,3 +1,5 @@
+// Copyright © 2022 Ory Corp
```

Comment:
Why don't we put the full declaration on each file as per Apache License?

```
   Copyright 2022 Ory Corp

   Licensed under the Apache License, Version 2.0 (the "License");
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at

       http://www.apache.org/licenses/LICENSE-2.0

   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
```

Reply:
@hperl Thanks for your suggestion. This question is still open. Please find the discussion around the license headers in https://github.com/ory-corp/general/issues/602. I went with the shortest possible option primarily because it is short. The headers can be changed later as well.

Reply:
@hperl FYI based on your suggestion I am now including the license name, but abbreviated in a standard format so that it is machine readable.